### PR TITLE
[DOC] Remove `..figure` directive in examples

### PR DESCRIPTION
**Pull request by skoudoro**
Sphinx gallery generate its own figure directive. So, currently our `..figure` directive are not processed and shows unfound files in our tutorials which is confusing. 

Furthermore, I added `.. rst-class:: centered small fst-italic fw-semibold` directive to simulate and not lose some captions.

### CODE REVIEW

**Comment by skoudoro** [comment]
Ok, this PR is ready to go.

is there any comments/suggestions ? I will go ahead and merge this tomorrow at noon if no comments. 

This cleaning was needed and more to come after the release.

File: doc/examples/bundlewarp_registration.py [hunk 1]
```diff
@@ -24,11 +25,10 @@
                                  viz_displacement_mag)
 from time import time
 
-"""
-Let's download and loaf two uncinate fasciculus bundles in the left hemisphere
-of the brain (UF_L) available here:
-https://figshare.com/articles/dataset/Test_Bundles_for_DIPY/22557733
-"""
+###############################################################################
+# Let's download and loaf two uncinate fasciculus bundles in the left
```
Comment (arokem) [inline doc/examples/bundlewarp_registration.py:29]:
```suggestion
# Let's download and load two uncinate fasciculus bundles in the left
```

File: doc/examples/denoise_mppca.py [hunk 2]
```diff
@@ -12,10 +12,11 @@
 The basic idea behind the PCA-based denoising algorithms is to remove the
 components of the data that are classified as noise. The Principal Components
 classification can be performed based on prior noise variance estimates
-[Manjon2013]_ (see :ref:`denoise_localpca`) or automatically based on the
-Marcenko-Pastur distribution [Veraa2016a]_. In addition to noise
-suppression, the PCA algorithm can be used to get the standard deviation of
-the noise [Veraa2016b]_.
+[Manjon2013]_
+(see :ref:`denoise_localpca<sphx_glr_examples_built_preprocessing_denoise_localpca.py>`)  # noqa E501
+or automatically based on the Marcenko-Pastur distribution [Veraa2016a]_.
```
Comment (arokem) [inline doc/examples/denoise_mppca.py:17]:
```suggestion
or automatically based on the Marchenko-Pastur distribution [Veraa2016a]_.
```

File: doc/examples/denoise_mppca.py [hunk 3]
```diff
@@ -72,33 +70,31 @@
...
-
-The PCA denoising using the Marcenko-Pastur distribution can be performed by
-calling the function ``mppca``:
-"""
+###############################################################################
+# As one can see from its shape, the selected data contains a total of 67
+# volumes of images corresponding to all the diffusion gradient directions
+# of the selected b-values.
+#
+# The PCA denoising using the Marcenko-Pastur distribution can be performed by
```
Comment (arokem) [inline doc/examples/denoise_mppca.py:78]:
```suggestion
# The PCA denoising using the Marchenko-Pastur distribution can be performed by
```

File: doc/examples/denoise_mppca.py [hunk 4]
```diff
@@ -72,33 +70,31 @@
...
-"""
+###############################################################################
+# Internally, the ``mppca`` algorithm denoises the diffusion-weighted data
+# using a 3D sliding window which is defined by the variable patch_radius.
+# In total, this window should comprise a larger number of voxels than the
+# number of diffusion-weighted volumes. Since our data has a total of 67
+# volumes, the patch_radius is set to 2 which corresponds to a 5x5x5 sliding
+# window comprising a total of 125 voxels.
+
+# To assess the performance of the Marcenko-Pastur PCA denoising algorithm,
```
Comment (arokem) [inline doc/examples/denoise_mppca.py:95]:
```suggestion
# To assess the performance of the Marchenko-Pastur PCA denoising algorithm,
```

File: doc/examples/denoise_mppca.py [hunk 5]
```diff
@@ -196,57 +184,48 @@
...
+###############################################################################
+# .. rst-class:: centered small fst-italic fw-semibold
+#
+# In the above figure, the DKI maps obtained from the original data are shown
+# in the upper panels, while the DKI maps from the denoised data are shown in
+# the lower panels. Substantial improvements in measurement robustness can be
+# visually appreciated, particularly for the FA and MK estimates.
+#
+#
+# Noise standard deviation estimation using the Marcenko-Pastur PCA algorithm
```
Comment (arokem) [inline doc/examples/denoise_mppca.py:196]:
```suggestion
# Noise standard deviation estimation using the Marchenko-Pastur PCA algorithm
```

File: doc/examples/denoise_mppca.py [hunk 6]
```diff
@@ -196,57 +184,48 @@
...
-   :align: center
-
-The above figure shows that the Marcenko-Pastur PCA algorithm computes a 3D
-spatial varying noise level map. To obtain the mean noise std across all
-voxels, you can use the following lines of code:
-"""
+###############################################################################
+# .. rst-class:: centered small fst-italic fw-semibold
+#
+# The above figure shows that the Marcenko-Pastur PCA algorithm computes a 3D
```
Comment (arokem) [inline doc/examples/denoise_mppca.py:218]:
```suggestion
# The above figure shows that the Marchenko-Pastur PCA algorithm computes a 3D
```

File: doc/examples/denoise_patch2self.py [hunk 7]
```diff
@@ -129,65 +134,57 @@
...
-             medicine, 65(2), pp.480-491.
-
-.. include:: ../links_names.inc
-
-"""
+###############################################################################
+# Lastly, one can also use Patch2Self in batches if the number of gradient
+# directions is very high (>=200 volumes). For instance, if the data has 300
+# volumes, one can split the data into 2 batches, (150 directions each) and
+# still get the same denoising performance. One can simply run Patch2Self
```
Comment (arokem) [inline doc/examples/denoise_patch2self.py:154]:
```suggestion
# still get the same denoising performance. One can run Patch2Self
```

**Review by arokem — APPROVED** [review]
Looks good to me! I spotted a couple of typos along the way, but that doesn't mean there aren't any other ones :-)

**Comment by skoudoro** [comment]
thank you for the review @arokem!

For sure there is other ones, like old functions a simpler way to do stuff in many tutorials. We will continue to update that.during the year.

**Comment by skoudoro** [comment]
ps: building the doc in parallel, just waiting the build to finish before merging this PR.

**Pull request merged**